cardano-node/app/wallet-client.hs

```diff
@@ -12,10 +12,10 @@ import           Cardano.Node.Configuration.Types (CardanoConfiguration (..),
                                                    CardanoEnvironment (..))
 import           Cardano.Node.Features.Logging (LoggingCLIArguments (..),
                                                 LoggingLayer (..),
-                                                createLoggingFeature,
-                                                loggingParser)
-import           Cardano.Prelude hiding (option)
+                                                createLoggingFeature
+                                                )
```

Comment:
Doesn't matter, really, but perhaps this ")" should be on the same line as `createLoggingFeature`.

Reply:
I actually need to figure out some formatting conventions :thinking: but I will also follow up on this in a later PR.

Reply:
I suggest using `stylish-haskell`, that already takes care of the imports.